tests/test_trainer.py

```diff
@@ -299,3 +299,19 @@ def test_trainer_iterable_dataset(self):
         trainer = Trainer(model=model, args=training_args, train_dataset=train_dataset)
         loader = trainer.get_train_dataloader()
         self.assertIsInstance(loader, torch.utils.data.DataLoader)
+
+    @require_non_multigpu
```

Comment:
This flag won't be necessary here, as the test will work equally well on a multigpu setup.

Reply:
OK. I kind copy paste `test_number_of_steps_in_training` which has `@require_non_multigpu`, since I am not familar with pytorch when it comes with distributed stuffs. I will remove.